app/i18n/en.json

```diff
@@ -71,16 +71,21 @@
   "receiveScreen": {
     "shareYourAddress": "Share your address"
   },
-  "RegistrationScreen": {
-    "likerIdHint": "Allow 7-20 lowercase letters, numbers, dash and underscore only",
-    "likerIdLabel": "Create your Liker ID",
-    "shouldSignInOnDesktop": "Please retry after signing in to http://like.co on desktop"
+  "ReferralScreen": {
+    "DescriptionLabel": "分享你的推薦鏈結給朋友，大家同獲奬賞，同時讓讀者追縱自己的更新。趕快邀請朋友下載 Liker Land 吧！",
+    "HeaderTitle": "推薦奬賞",
+    "Title": "推薦愈多，奬金愈多"
```

Comment:
This is en.json, is this placeholder waiting for translate?

Reply:
Yes, it is placeholder